cylc/flow/scripts/validate.py

```diff
@@ -89,8 +93,14 @@
         dest="profile_mode",
         sources={'validate'}
     ),
+    OptionSettings(
+        ["-u", "--run-mode"], help="Validate for run mode.", action="store",
+        default="live", dest="run_mode",
+        choices=['live', 'dummy', 'simulation']
+    ),
```

Comment:
Now `cylc validate` has two copies of this option :confused:

```console
$ cylc val --help
...
  -u RUN_MODE, --run-mode=RUN_MODE
                        Validate for run mode.
  -m STRING, --mode=STRING
                        Run mode: live, dummy, simulation (default live).
...
```

> **Context:** I think you were changing this because the previous value conflicted with another option?

Reply:
Yes, that's wrong! Merge/rebase error.